pubsub/google/cloud/pubsub/topic.py

```diff
@@ -435,12 +462,27 @@ def publish(self, message, **attrs):
 
         :type attrs: dict (string -> string)
         :param attrs: key-value pairs to send as message attributes
+
+        :rtype: None
+        :returns: None
         """
         self.topic._timestamp_message(attrs)
         self.messages.append(
             {'data': message,
              'attributes': attrs})
 
+        # If too much time has elapsed since the first message
+        # was added, autocommit.
+        if self._max_interval < self.INFINITY:
```

Comment:
Why is this check needed? No `float` will ever be `> infinity` in the next check